### PR TITLE
added FAQ component

### PR DESCRIPTION
**Pull request by toucham**
**Changes:**
- added static content according to [design](https://app.zeplin.io/project/622bcca0bb247263a8fe99bf/screen/6234d6c5f30b7797105d7620)
- currently rendered in `LandingPage.tsx`, will be later moved when the page is done

https://user-images.githubusercontent.com/52475722/159336209-67dabd19-78cb-416e-b168-305df2488403.mov

### CODE REVIEW

**Comment by Ingi-Hong** [comment]
Looks awesome, I guess all that needs to be done is to change the font size and the icon. Let me know if you want me to do those things toucham, I have time probably tonight or tomorrow. 

https://ant.design/components/collapse/#header
https://codepen.io/pen?&editors=001

**Comment by IanSaucy** [comment]
Deployed URL: https://b7dd066c.uncounteddeath.pages.dev/

File: src/components/FAQ.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      'Excess mortality refers to differences in mortality relative to what would have been expected in the absense of the COVID-19 pandemic. These estimates provide a useful measure of the total mortality burden associated with COVID-19.',
+  },
+  {
+    title: 'What are the assigned and unassigned death rates?',
+    content:
+      'When any individual dies, the reason is recorded on their death certificate. We would expect that most of the excess deaths are due to the pandemic, and so are recorded on the death certificate as a Covid death. This metric defines of the the total excess deaths, how many are officially recorded as being from Covid.',
+  },
+  {
+    title: 'What are fitted and total death rates?',
+    content:
```
Comment (IanSaucy) [inline src/components/FAQ.tsx:24]: Could you move the content into the `/content` directory?

File: src/styles/FAQ.module.less [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.FAQ {
+  padding: 0.5rem 10rem;
+}
```
Comment (IanSaucy) [inline src/styles/FAQ.module.less:3]: Is there a difference in `.less` vs `.module.less` files?

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Looks super good, just two small things (:

File: src/styles/FAQ.module.less (hunk 2 above)
Comment (toucham) [reply src/styles/FAQ.module.less:3]: None, will move css into `.module.less` to `.less`

File: src/components/FAQ.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+const FAQ: React.FC = () => {
+  return (
+    <div className="faq-container">
+      <Collapse
+        expandIcon={({ isActive }) => (isActive ? <MinusOutlined /> : <PlusOutlined />)}
+        expandIconPosition="right"
+      >
+        {FAQ_data.map((f, i) => (
+          <Collapse.Panel key={i} header={<Typography.Text strong>{f.title}</Typography.Text>}>
```
Comment (IanSaucy) [inline src/components/FAQ.tsx:15]: Title text size is wrong, can you try one of the other "levels" of Typography as provided by antd to get it closer to the Zeplin designs.

File: src/components/FAQ.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            <Typography.Text>{f.content}</Typography.Text>
```
Comment (IanSaucy) [inline src/components/FAQ.tsx:16]: Same with this text. Line height also needs to be adjusted.

File: src/content/FAQContent.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
+export interface IFAQ {
```
Comment (IanSaucy) [inline src/content/FAQContent.ts:1]: This is great, thank you!

**Review by IanSaucy — CHANGES_REQUESTED** [review]
Just a few small changes in regards to the UX designs.

File: src/components/FAQ.tsx (hunk 3 above)
Comment (toucham) [reply src/components/FAQ.tsx:15]: There is no levels for `Typography.Text` so instead I will directly change the `font-size` in less

File: src/components/FAQ.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+import { Collapse, Typography } from 'antd';
+import '../styles/FAQ.less';
+import { FAQ_data } from '../content/FAQContent';
+import { MinusOutlined, PlusOutlined } from '@ant-design/icons';
+
+const FAQ: React.FC = () => {
+  return (
+    <div className="faq-container">
+      <Collapse
+        expandIcon={({ isActive }) => (isActive ? <MinusOutlined /> : <PlusOutlined />)}
```
[the remark this answers is not in the document]
Comment (toucham) [reply src/components/FAQ.tsx:11]: The antd icon can't be bold but instead I'll just use the html & css on zeplin with a few tweaks

**Review by IanSaucy — APPROVED** [review]
super great!

**Pull request merged**